dlt/common/configuration/specs/gcp_credentials.py

```diff
@@ -64,6 +64,15 @@ def to_gcs_credentials(self) -> Dict[str, Any]:
             ),
         }
 
+    def to_object_store_rs_credentials(self) -> Dict[str, str]:
+        """
+        Dict of keyword arguments that can be passed to `object_store` Rust crate.
+        Delegates default GCS credential handling to `object_store` Rust crate.
+        """
+        if isinstance(self, CredentialsWithDefault) and self.has_default_credentials():
```

Comment:
FYI; this could be also oauth2 credential. still passing of defaults will work so this is OK. dict however will have a wrong format. but I really doubt that anyone will try oauth2 here